holidays/countries/micronesia.py

```
@@ -0,0 +1,209 @@
+
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase, SAT_TO_PREV_FRI, SUN_TO_NEXT_MON
+
+
+class Micronesia(ObservedHolidayBase, ChristianHolidays, InternationalHolidays):
+    """Micronesia Holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_the_Federated_States_of_Micronesia>
+        * <https://www.timeanddate.com/holidays/micronesia/>
+        * <https://web.archive.org/web/20241213112548/http://www.fsmlaw.org/fsm/code/PDF/FSMCA2014Tit01.pdf>
```

Pylint (3.3.7)

[convention] Line too long (108/100)

(C0301)